browser/resources/settings/brave_overrides/basic_page.js

```diff
@@ -199,18 +199,22 @@ RegisterPolymerTemplateModifications({
           prefs: '{{prefs}}'
         }
       ))
-      const sectionRewards = document.createElement('template')
-      sectionRewards.setAttribute('is', 'dom-if')
-      sectionRewards.setAttribute('restamp', true)
-      sectionRewards.setAttribute('if', '[[showPage_(pageVisibility.rewards)]]')
-      sectionRewards.content.appendChild(createSectionElement(
-        'rewards',
-        'braveRewards',
-        'settings-brave-rewards-page',
-        {
-          prefs: '{{prefs}}'
-        }
-      ))
+      const isRewardsEnabled = loadTimeData.getBoolean('isBraveRewardsEnabled')
```

Comment:
This may be a little confusing to us down the line, because in our code base "rewards enabled" typically means "rewards is toggled on". Maybe we should name this something more like `isBraveRewardsAllowed` or `isBraveRewardsPermitted`?

Reply:
I agree. Other options to consider:

- `isBraveRewardsAvailable`
- `isBraveRewardsSupported`

It should probably also be consistent with whatever is chosen for `chrome.braveRewards`.

Reply:
Right, maybe use the same terminology as in NTP: `isBraveRewardsSupported`

Reply:
Updated in 7c47c0473fe48b44ee7dcef53fcbfe2510df602b and 32b27410bf3f73824041af3fc10ab7e240a93d15